deployment/legacy-migration/maintenance-worker/migration/neo4j/badges.cql

```diff
@@ -1,4 +1,4 @@
-CALL apoc.load.json('file:/tmp/mongo-export/splits/current-chunk.json') YIELD value as badge
+CALL apoc.load.json("file:${IMPORT_CHUNK_PATH_CQL}") YIELD value as badge
```

Comment:
I don't think you can expand variables like this. It's a `.cql` file, not a bash script.

Reply:
You can if you use magic like I do here: https://github.com/Human-Connection/Human-Connection/pull/697/files#diff-c6d49070ce711f563a75e723b1e8682cR14